Competition2018/src/main/java/competition/commandgroups/DynamicScoreOnScaleCommandGroup.java

```diff
@@ -0,0 +1,40 @@
+package competition.commandgroups;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.autonomous.AutonomousDecisionSystem;
+import competition.subsystems.elevator.ElevatorSubsystem;
+import competition.subsystems.elevator.commands.SetElevatorTargetHeightCommand;
+import competition.subsystems.gripperintake.commands.GripperEjectCommand;
+import competition.subsystems.wrist.commands.SetWristAngleCommand;
+import openrio.powerup.MatchData.GameFeature;
+import xbot.common.command.BaseCommandGroup;
+import xbot.common.subsystems.drive.PurePursuitCommand;
+
+public class DynamicScoreOnScaleCommandGroup extends BaseCommandGroup {
```

Comment:
Ideally Scale/Switch commands would be generated by a factory (or maybe controlled by some more properties), but we don't have a competition factory already set up, and I want to make sure the autonomous command chooser shows really explicit information